.github/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+<!--- When creating a new PR, make sure to request a review and tag related issues -->
```

Comment:
I think you need to be a contributor in order to be able to request reviews, so it's not something that anyone can do.

Reply:
Good point, removed that line.